package.json

```diff
@@ -22,8 +22,8 @@
   },
   "main": "./lib/index.js",
   "dependencies": {
-    "array-changes": "1.3.1",
-    "array-changes-async": "2.2.1",
+    "array-changes": "unexpectedjs/array-changes#e2385e5b3130e7c715dee6d77489c78565833bcb",
+    "array-changes-async": "bruderstein/array-changes-async#8c965f125be6f45604ef764a7750b4c23eafd9f9",
```

Comment:
We of cause needs these released before this can be merged. But I guess we are waiting on an answer from @bruderstein - right?